public/components/FeatureAnywhereContextMenu/CreateAnomalyDetector/AddAnomalyDetector.tsx

```diff
@@ -0,0 +1,823 @@
+                >
+                  {[
+                    {
+                      id: 'add-anomaly-detector__create',
+                      label: 'Create new detector',
+                      value: 'create',
+                    },
+                    {
+                      id: 'add-anomaly-detector__existing',
+                      label: 'Associate existing detector',
+                      value: 'existing',
+                    },
+                  ].map((option) => (
+                    <EuiCheckableCard
+                      {...{
+                        ...option,
+                        key: option.id,
+                        name: option.id,
+                        checked: option.value === mode,
+                        onChange: () => setMode(option.value),
+                      }}
+                    />
+                  ))}
+                </EuiFormFieldset>
+                <EuiSpacer size="m" />
+                {mode === 'create' && (
+                  <div className="create-new">
+                    <EuiText size="xs">
+                      <p>
+                        Create and configure an anomaly detector to
+                        automatically detect anomalies in your data and to view
+                        real-time results on the visualization.{' '}
+                        <a
+                          href="https://opensearch.org/docs/latest/observing-your-data/ad/index/"
+                          target="_blank"
+                        >
+                          Learn more <EuiIcon type="popout" />
+                        </a>
+                      </p>
+                    </EuiText>
+                    <EuiSpacer size="m" />
+                    <div className="create-new__title-and-toggle">
+                      <EuiTitle size="xxs">
+                        <h4>
+                          <EuiIcon
+                            type="visLine"
+                            className="create-new__title-icon"
+                          />
+                          {title}
+                        </h4>
+                      </EuiTitle>
+                      <EuiSwitch
+                        label="Show visualization"
+                        checked={isShowVis}
+                        onChange={() => setIsShowVis(!isShowVis)}
+                      />
+                    </div>
+                    <div
+                      className={`create-new__vis ${
+                        !isShowVis && 'create-new__vis--hidden'
+                      }`}
+                    >
+                      <EuiSpacer size="s" />
+                      <EmbeddablePanel
+                        embeddable={embeddable}
+                        getActions={() => Promise.resolve([])}
+                        inspector={{ isAvailable: () => false }}
+                        hideHeader
+                        isRetained
+                        isBorderless
+                      />
```

Comment:
You can use the embeddable plugin to use `PanelComponent` instead of `EmbeddablePanel` which requires a bunch more props. For example, see [here](https://github.com/ohltyler/OpenSearch-Dashboards/blob/view-events-2/src/plugins/vis_augmenter/public/view_events_flyout/components/base_vis_item.tsx)

Reply:
I was referred by Ashish to use the `EmbeddableRenderer` instead here like shown [here](https://github.com/opensearch-project/alerting-dashboards-plugin/pull/553). Do you have a preference for that vs `PanelComponent`

Reply:
Updated it with EmbeddableRenderer in the new revision

Reply:
Oh cool - wasn't aware of that. That's a better change

Reply:
Thanks for updating!